App/src/screen/Login/Login.js

```diff
@@ -17,6 +19,16 @@ const Login = ({navigation}) => {
   const login = () => {
     console.log(state);
```

Comment:
Delete the console.log(), you can use Alert.alert()

Reply:
using console.log for compatiblity develop in ios and android but deleting this code

Reply:
resolve problems and fix